src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor.cs

```diff
@@ -1,5 +1,46 @@
-﻿namespace Bit.Platform.WebSite.Web.Components;
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.Routing;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using Bit.Platform.WebSite.Web.Shared;
+
+namespace Bit.Platform.WebSite.Web.Components;
 
 public partial class GettingStarted
 {
+    [Inject] public NavigationManager NavigationManager { get; set; }
+    [Inject] public IJSRuntime JSRuntime { get; set; }
+    public string CurrentUrl { get; set; }
+    public ElementReference GettingStartedElement { get; set; }
+
+    protected override void OnInitialized()
+    {
+        CurrentUrl = NavigationManager.Uri.Replace(NavigationManager.BaseUri + "project-templates/todo-template/getting-started", "", StringComparison.Ordinal);
```

Comment:
please introduce a new const for this part and use it in both places:
`NavigationManager.BaseUri + "project-templates/todo-template/getting-started"`